tools/doxygen/doxygen.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+LAMMPSBASE=${PWD} ;
```

Comment:
All the `;` are not needed.

Reply:
Not needed but syntactically correct. Makes code argumentation more clear. Bash likes them.